### PR TITLE
Automated setup of jenkins and added it to the docker-compose.yml

### PR DESCRIPTION
**Pull request by garethgeorge**
@EasterAndJay @wpeery 

This should automate the setup for jenkins. At the moment jenkins is configured to be reset after every launch. I think this is the desired behavior for the time being.

### CODE REVIEW

**Comment by garethgeorge** [comment]
note travis-ci checks fail as travis will need to be updated to work with dockerized deployment.

File: docker-compose.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ services:
     command: sh ./docker/anacapa/docker-entrypoint.sh
     volumes:
       - .:/anacapa
+      - /anacapa/tmp/
```
Comment (garethgeorge) [inline docker-compose.yml]: this is supposed to prevent mounting /anacapa/tmp

File: docker/anacapa/docker-entrypoint.sh [hunk 2]
```diff
@@ -1,7 +1,5 @@
-echo "Running Database Setup & Maintenance"
-pwd
 cd anacapa/
-rm tmp/
+rm -rf tmp/
```
Comment (garethgeorge) [inline docker/anacapa/docker-entrypoint.sh]: if that doesn't work we also just remove the directory as it creates issues.

**Review by jaeaster — APPROVED** [review]

**Comment by jaeaster** [comment]
closes #45 and #44

**Pull request merged**